projects/desktop/src/components/LoginScreen/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import {BigTitle, LoginLanguageChooser, Logo} from './styles'
+import { Container, LeftSide, RightSide } from './parts'
+import { Form as FinalForm } from 'react-final-form'
+import { Form } from './form'
+import { useTranslation } from 'react-i18next'
+import logo from '../../assets/img/logo-small.png'
+
+export const LoginScreen = (): JSX.Element => {
+    const [t] = useTranslation()
+
+    const getTitleContent = (): {__html: string} => {
+        return {__html: t('login.title')};
+    }
+
+    return (
+        <Container>
+            <LeftSide>
+                <Logo src={logo} />
+                <LoginLanguageChooser />
+                <BigTitle dangerouslySetInnerHTML={getTitleContent()} />
+
+                <FinalForm
+                    onSubmit={values => console.log(values)}
```

Comment:
the `console.log(values)` has to stay here ?

Reply:
Yes while the backend is not done